src/Sylius/Bundle/UserBundle/DependencyInjection/SyliusUserExtension.php

```diff
@@ -34,13 +50,247 @@ public function load(array $config, ContainerBuilder $container)
 
         $loader->load(sprintf('driver/%s.xml', $config['driver']));
 
-        $this->registerResources('sylius', $config['driver'], $config['resources'], $container);
+        $this->registerResources('sylius', $config['driver'], $this->resolveResources($config['resources'], $container), $container);
 
         $loader->load('services.xml');
 
-        $container->setParameter('sylius.user.resetting.token_ttl', $config['resetting']['token']['ttl']);
-        $container->setParameter('sylius.user.resetting.token_length', $config['resetting']['token']['length']);
-        $container->setParameter('sylius.user.resetting.pin_length', $config['resetting']['pin']['length']);
-        $container->setParameter('sylius.user.verification.token_length', $config['verification']['token']['length']);
+        $this->createServices($config['resources'], $config['driver'], $container);
+    }
+
+    /**
+     * @param array $resources
+     * @param ContainerBuilder $container
+     *
+     * @return array
+     */
+    private function resolveResources(array $resources, ContainerBuilder $container)
+    {
+        $container->setParameter('sylius.user.users', $resources);
+
+        $resolvedResources = [];
+        foreach ($resources as $variableName => $variableConfig) {
+            foreach ($variableConfig as $resourceName => $resourceConfig) {
```

Comment:
I will move this method to ``AbstractResourceExtension`` because it is used not only in this extension.